src/cdk/drag-drop/directives/config.ts

```diff
@@ -43,4 +43,5 @@ export interface DragDropConfig extends Partial<DragRefConfig> {
   listAutoScrollDisabled?: boolean;
   listOrientation?: DropListOrientation;
   zIndex?: number;
+  previewContainer?: 'global' | 'parent';
```

Comment:
Why isn't this option type `PreviewContainer`?

Reply:
This is the interface for the global config object. It didn't seem like an `HTMLElement` would make sense there, given that it's configured before the app is initialized.